### PR TITLE
"Simple TCR" outline

### PR DESCRIPTION
**Pull request by Destiner**

### CODE REVIEW

File: Outlines/simple-tcr.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Proposed title
+Simple Token-Curated Registry
```
Comment (ChrisChinchilla) [inline Outlines/simple-tcr.md]: @Destiner Titles should be more descriptive and explain what the reader is going to gain from reading this, so how about 

```suggestion
Creating incentivized lists with a Token-Curated Registry
```

or something like this…

File: Outlines/simple-tcr.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Structure
+1. What is Token-Curated Registry
```
Comment (ChrisChinchilla) [inline Outlines/simple-tcr.md]:
```suggestion
1. What is a Token-Curated Registry
```

File: Outlines/simple-tcr.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+2. Examples of TCRs
+3. Creating a TCR
+	1. Smart contract
+	2. Front-end
+4. Limitations and improvement ideas
+5. Wrap-up
+
+## Motivation
+One of the biggest powers of Ethereum is the ability to easily program a system with a desired behaviour of its users. Different patterns of such mechanisms were discovered and then implemented. Token-curated registry is one such mechanism. In short, it allows to create a network of incentivised curators to create a list based on some criteria. Such a flexible tool opens a lot of possibilities for aspiring dapp developers.
```
Comment (ChrisChinchilla) [inline Outlines/simple-tcr.md:14]: One thing here, when it comes to the final piece, make sure you don't use words like, 'easy', "just" etc. These are called "Weasel words", if you search you'll find examples, or I can recommend tools that help you spot them.

File: Outlines/simple-tcr.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Proposed title
+Simple Token-Curated Registry
+
+## Structure
+1. What is Token-Curated Registry
+2. Examples of TCRs
```
Comment (ChrisChinchilla) [inline Outlines/simple-tcr.md:6]: Let's make sure this has some good understandable examples, as reading through many posts I am still struggling to understand quite why a TCR is useful.

File: Outlines/simple-tcr.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+	2. Front-end
+4. Limitations and improvement ideas
+5. Wrap-up
+
+## Motivation
+One of the biggest powers of Ethereum is the ability to easily program a system with a desired behaviour of its users. Different patterns of such mechanisms were discovered and then implemented. Token-curated registry is one such mechanism. In short, it allows to create a network of incentivised curators to create a list based on some criteria. Such a flexible tool opens a lot of possibilities for aspiring dapp developers.
+
+## Who should read this
+Anyone who is interested in creating decentralized applications as well as those who want to learn more about game theory and mechanism design.
+The article will include code samples in Solidity and JS and will require a moderate technical background.
```
Comment (ChrisChinchilla) [inline Outlines/simple-tcr.md:18]: Good! Let's keep the Blockchain hype low and the practical how to build it high :)

**Comment by ChrisChinchilla** [comment]
@Destiner Couple of small comments, once we're done with those I'll create the bounty and we can get started.

File: Outlines/simple-tcr.md (hunk 3 above)
Comment (Destiner) [reply Outlines/simple-tcr.md:14]: Makes sense, will keep in mind in the future.

File: Outlines/simple-tcr.md (hunk 4 above)
Comment (Destiner) [reply Outlines/simple-tcr.md:6]: Right. I was thinking about starting with a simple made-up example, and then go in-depth with some existing TCR project.

File: Outlines/simple-tcr.md (hunk 1 above)
Comment (Destiner) [reply Outlines/simple-tcr.md]: What do you think of
1) Incentivizing high-quality curation with a Token-Curated Registry
2) Incentivizing item selection with a Token-Curated Registry?

**Comment by Destiner** [comment]
@ChrisChinchilla looks good. I think once we finalize on title, everything should be set up.

File: Outlines/simple-tcr.md (hunk 1 above)
Comment (ChrisChinchilla) [reply Outlines/simple-tcr.md]: @Destiner Number 1.

Comment (Destiner) [reply Outlines/simple-tcr.md]: Changed the title.

**Comment by ChrisChinchilla** [comment]
@Destiner Right, let's do it, I'll set up the bounty now, and "start work" when you're ready.

**Pull request merged**